src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbyplaintextcommentfilter/InputSuppressWithNearbyPlainTextCommentFilterCommentFormat.css

```diff
@@ -0,0 +1,113 @@
+/*
+SuppressWithNearbyPlainTextCommentFilter
+commentFormat = cs (\\w+)
+checkFormat = (default).*
+messageFormat = (default)(null)
+idFormat = (default)(null)
+influenceFormat = (default)0
+
+com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck
```

Comment:
and original issue was about lineLenght - https://github.com/checkstyle/checkstyle/issues/7064

Reply:
Tests already use `LineLengthCheck`. Just removed `FileTabCharacterCheck` and put `RegexpSinglelineCheck` in its place since we need 2 checks for full test coverage.

Reply:
Done.